src/js/components/CheckBoxGroup/CheckBoxGroup.js

```diff
@@ -0,0 +1,99 @@
+import React, { forwardRef, useContext, useMemo } from 'react';
+
+import { Box } from '../Box';
+import { CheckBox } from '../CheckBox';
+import { FormContext } from '../Form/FormContext';
+
+export const CheckBoxGroup = forwardRef(
+  (
+    {
+      checked: checkedProp,
+      disabled: disabledProp,
+      gap = 'small', // consistent with RadioButtonGroup default
+      labelKey,
+      valueKey,
+      onChange,
+      options: optionsProp,
+      name,
+      ...rest
+    },
+    ref,
+  ) => {
+    const formContext = useContext(FormContext);
+
+    // In case option is a string, normalize it to be an object
+    const options = useMemo(
+      () =>
+        optionsProp.map(option => {
+          return typeof option === 'string'
+            ? {
+                disabled: disabledProp,
+                id: option,
+                label: option,
+              }
+            : option;
+        }),
+      [optionsProp, disabledProp],
+    );
+
+    // 'checked' is an array of checked valueKeys
+    const [checked, setChecked] = formContext.useFormContext(
+      name,
+      checkedProp,
+      '',
```

Comment:
I don't think the initial value would be `''`. I think it can be omitted, or maybe `[]`.

Reply:
It worked with `[]` better than omitting.